### DISCUSSION TITLE
osu! for other Linux distributions

### DISCUSSION THREAD

**Discussion by EPICNESS2008** [ppy/osu#30571] [Q&A] [answered]
Hello, is there a .deb port of osu! that I can get?

**Comment by sineplusx** [comment]
The official releases use the AppImage format to serve all kinds of Linux distros. Is the AppImage not working for you?

**Reply by EPICNESS2008** [reply]
I don't know how to install an appimage. That's the reason why I'm asking if there is a Debian distrib

**Reply by sineplusx** [reply, marked as answer]
Well, not having to install it is the whole point of the AppImage format. I dont know any Debian-based package for osu.

If you just want to have it be somewhere permanently, you could also look into the [Flatpak version](https://flathub.org/apps/sh.ppy.osu) or use some tool like [AppImageLauncher](https://github.com/TheAssassin/AppImageLauncher).

**Reply by bdach** [reply]
We're not doing distro specific releases. We're doing two separate 'portable' linux distribution formats anyway as mentioned above.

**Comment by baharsah** [comment]
can you describe your distro you want to run it to (e.g. ubuntu) ?
for "How to install an appimage" section, you don't need to install it. just run it as executable file.
if you cant run it trough, you can change the permission of that file to executable file.

**Reply by EPICNESS2008** [reply]
Uhh, chromeOS, which I found out runs Gentoo. But, I found out that left-clicking a Debian port of an app, it would give me the option to install it with Linux.

**Reply by baharsah** [reply]
we don't have a version of the .deb package for this case yet.
have you try to run it (the .appimage) via terminal as an executable file?

**Reply by EPICNESS2008** [reply]
How do I make an appimage an executable?

**Reply by 12beesinatrenchcoat** [reply]
You use the terminal for that:
```sh
chmod +x [path to file]
```
